src/Sulu/Bundle/MediaBundle/Tests/Functional/Entity/MediaDataProviderRepositoryTest.php

```diff
@@ -651,10 +654,8 @@ public static function provideFindByFiltersWithAudienceTargeting()
         ];
     }
 
-    /**
-     * @dataProvider provideFindByFiltersWithAudienceTargeting
-     */
-    public function testFindByFiltersWithAudienceTargeting($targetGroupIndex, $expectedIndexes): void
+    #[\PHPUnit\Framework\Attributes\DataProvider('provideFindByFiltersWithAudienceTargeting')]
+    public function testFindByFiltersWithAudienceTargeting(?int $targetGroupIndex, array $expectedIndexes): void
```

Comment:
test method changes should be done on 2.5 branch if possible and non breaking changes